src/index.ts

```diff
@@ -0,0 +1,80 @@
+import { TASK_COMPILE_GET_SOURCE_PATHS } from "@nomiclabs/buidler/builtin-tasks/task-names";
+import { task } from "@nomiclabs/buidler/config";
+import { BuidlerConfig } from "@nomiclabs/buidler/types";
+import { ensureDir, readFile, writeFile } from "fs-extra";
+import { defaults } from "lodash";
```

Comment:
`defaults(a, b)` is the same than `{...a, ...b}`. There's no need to install `lodash` here.

Reply:
Sorry, I meant `{...b, ...a}`